package.json

```diff
@@ -62,7 +62,8 @@
     "example:clean": "cd RNFBSDKExample && \\rm -fr yarn.lock node_modules ios/Podfile.lock && cd ..",
     "example:install": "cd RNFBSDKExample && yarn && cd ios && (pod install || true) && cd ../..",
     "example:devcopy": "yarn prepare && cp -rv android ios lib src types *.podspec RNFBSDKExample/node_modules/react-native-fbsdk-next/",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "jest": "jest"
```

Comment:
Very interesting, how does CI go when you change package.json line 55 to include ` && yarn jest` so this is exercised?

Reply:
Hey @mikehardy, I think we can put this on line 55 to run with the test script, I'll try it.

Reply:
it's working 😄

Reply:
2fe289a7d2ec3d5bc9583fb573574dd0b0c740fc

Reply:
fantastic!